packages/fast-usdc-deploy/src/index.js

```diff
@@ -0,0 +1,4 @@
+export * from './utils/chain-policies.js';
+export * from './utils/config-marshal.js';
+export * from './utils/core-eval.js';
+export * from './utils/deploy-config.js';
```

Comment:
again, my impression is that this was left out by design. I'm not in a good position to review this change